src/main/java/com/mrdb/controller/MainController.java

```diff
@@ -0,0 +1,46 @@
+package com.mrdb.controller;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.util.JSONPObject;
+import com.mrdb.entities.UserEntity;
+import com.mrdb.entities.UserEntityRepository;
+import org.apache.tomcat.util.json.JSONParser;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.HashMap;
+
+@RestController
+@RequestMapping("/api/users")
+public class MainController {
+    @Autowired private UserEntityRepository repo;
+
+    @PostMapping("/login")
+    public String login(@RequestBody UserEntity userDetails) {
+        String res = "Ok";
+        String userName = userDetails.getUserName();
+        String password = userDetails.getPassword();
+        UserEntity user = repo.findByUserName(userName);
+        if(null != user && password.equalsIgnoreCase(user.getPassword())) {
+            res = "Ok";
+        } else {
+            res = "Not Ok";
+        }
+
+        return res;
+    }
+
+    @PostMapping("/register")
+    public String register(@RequestBody UserEntity userDetails) {
+        String res = "Ok";
```

Comment:
At this point we’ve used the “Ok” string several times. It better to create these string literals as constant variables and use those constants in all of these places. That way, if you ever want to change the message text later on, you just can change it in one place instead of all these places.

Reply:
Will add a new variable success in application.properties in next sprint